docs/pages/demo/daterangepicker/index.mdx

```diff
@@ -13,9 +13,14 @@ import * as CustomRangeInputs from './CustomRangeInputs.example';
 
 <PageMeta component="DateRangePicker" />
 
-## Date picker
+## Date range picker
 
-[Date pickers](https://material.io/components/pickers/) let users select a date, or a range of dates. They should be suitable for the context in which they appear.
+> ⚠️⚠️ The date range picker is unstable, it's **not suitable** for usage in production. ⚠️⚠️
+
+> The date range picker will be made available in the coming month for production usage as part of a paid extension of the community edition (MIT license) of Material-UI.
+> This paid extension will include advanced components (rich data grid, date range picker, tree view drag & drop, etc.). Pricing plans will start at \$129/developer for early access.
```

Comment:
```suggestion
> This paid extension will include advanced components (rich data grid, date range picker, tree view drag & drop, etc.).
```
1. Do we need pricing here?
2. Is paid support for MIT components in or out? (It isn't mentioned here.)

Reply:
1. I think that it can help developers decide if they want to use the alpha anyway or move to alternatives.
2. How does the answer to this question impact the warning? It seems to be independent.

Reply:
2. You're right, it's incomplete - it doesn't mention support at all, so MIT doesn't matter one way or the other.

Reply:
So we should rephrase the previous sentence? 

```diff
-The date range picker will be made available in the coming month for production usage as part of a paid extension of the community edition (MIT license) of Material-UI.
+The date range picker will be made available in the coming month for production usage as part of a paid extension of the MIT licensed modules of Material-UI.
```